File: Python/pola-bintang.py
baris = int(input("Maskkan banyak baris: "))

for i in range(1, baris + 1):
    for j in range(1, i + 1):
        print("* ", end="")
    print()

print("============")

bintang = ""
for i in range(1, baris + 1):
    bintang += "* "
    print(bintang)

print("============")

for i in range(1, baris + 1):
    # spasi
    for j in range(baris - 1, i - 1, -1):
        print(" ", end="")

    # bintang
    for k in range(1, i + 1):
        print("* ", end="")

    print()

File: C++/pola-bintang.cpp
#include <iostream>

using namespace std;

int main()
{
    int baris;

    cout << "Masukkan banyak baris: ";
    cin >> baris;

    for (int i = 1; i <= baris; i++)
    {
        for (int j = 1; j <= i; j++)
        {
            cout << "* ";
        }
        cout << "\n";
    }

    cout << "===========\n";

    string bintang = "";
    for (int i = 1; i <= baris; i++)
    {
        bintang += "* ";
        cout << bintang << endl;
    }

    cout << "===========\n";

    for (int i = 1; i <= baris; i++)
    {
        // spasi
        for (int j = (baris - 1); j >= i; j--)
        {
            cout << " ";
        }

        // bintang
        for (int j = 1; j <= i; j++)
        {
            cout << "* ";
        }
        cout << "\n";
    }

    return 0;
}